src/js/fluxcontroller.js

```diff
@@ -202,6 +208,26 @@ define(function (require, exports, module) {
     });
 
     /**
+     * Initialize all action functions 
+     *
+     * @private
+     */
+    FluxController.prototype._initAllActions = function () {
+        Object.keys(actionIndex).forEach(function (actionModuleName) {
+            var actionModule = actionIndex[actionModuleName],
+                moduleActions = {};
+            Object.keys(actionModule)
+                .forEach(function (actionName) {
+                    if (actionModule[actionName].action) {
+                        actionModule[actionName].module = actionModuleName;
```

Comment:
I don't think we should modify the `.action` object. Instead, we should just keep this state in the `FluxController` instances, the same way we keep the locks and the name/function mappings.